packages/ui/src/global.css

```diff
@@ -1,5 +1,5 @@
 :root {
-  --color-primary: #011627;
+  --color-primary: #00B87C;
```

Comment:
they were actually just placeholders for now
feel free to edit the colours here

Reply:
i think these css vars may not be necessary anymore. i define the theme colors already using mantine

Reply:
feel free to leave them in or take them out